src/cargo/ops/cargo_new.rs

```diff
@@ -253,8 +253,7 @@ fn check_name(
     if restricted_names::is_non_ascii_name(name) {
         shell.warn(format!(
```

Comment:
I am wondering if this should be a warning or an error. To me, it seems like it should be an error but one of the [cargo team members](https://www.rust-lang.org/governance/teams/dev-tools#cargo) may have a different idea.

Reply:
Such a package rename still works. I feel like changing it to hard error is a bit rush at this moment. 

```toml
[dependencies]
hello = { path = "./你好", package = "你好" }
```